source/_integrations/switcher_kis.markdown

```diff
@@ -105,6 +105,10 @@ For Switcher Breeze the integration provides the following sensor:
 | --------------------- | ------------------------------------------------------ | ----------------- |
 | `Current temperature` | The current temperature in celsius                     | 25.0 °C           |
 
+## Switches
+
+For Switcher cover control devices (Switcher Runner, Switcher Runner S11, Switcher Runner S12), the integration allows you to control the cover physical controls also known as child lock state; ON means locked, and OFF means unlocked.
```

Comment:
The description here doesn't make sense, please align it with the approved PR which was closed: https://github.com/home-assistant/home-assistant.io/pull/36188

Reply:
It is done.